include/swift/AST/DeclAttr.def

```diff
@@ -862,7 +862,7 @@ DECL_ATTR(abi, ABI,
 DECL_ATTR_FEATURE_REQUIREMENT(ABI, ABIAttribute)
 
 DECL_ATTR(execution, Execution,
-  OnAbstractFunction,
+  OnAbstractFunction | OnSubscript | OnVar,
```

Comment:
What is the preferred place for these attributes, is it on the accessor or its storage declaration? I think making a decision now could save us from the situation we are in with global actor attributes...

Reply:
The storage is the preferred location. I agree it would be better to exclude accessors for now.